test/expect/TestScript.test_if_supertype.expect

```diff
@@ -0,0 +1,13 @@
+graph(%y.1 : Float(2, 2)
+      %z.2 : Long(2, 4)
+      %z.1 : Float(2, 4)) {
+  %3 : int = prim::Constant[value=1]()
+  %x : Float(2, 2), %y : Dynamic, %z : Dynamic = prim::If(%3)
```

Comment:
Isn't it the case that the third output of both branches has the same type? Why is it set to dynamic in this test?

Reply:
The third output has a different at::scalar type (a limitation of our current tensor type that will be changed)